src/test-utils.ts

```diff
@@ -0,0 +1,3 @@
+export const asMockedFn = <T extends (...args: unknown[]) => unknown>(fn: T): jest.MockedFunction<T> => {
```

Comment:
Taken from #3447 (though I swapped `any` for `unknown`).

Reply:
I've had bad luck using unknown for these very general helpers.  Unknown tells TS that we don't care about type-flow, and has the added benefit of treating "pretend we know" interactions with the unknown type as errors (handy for pass-thru fns, etc).  But here we DO care about type flow, so we need to use any in the extends clause.  I pair this with a //es-ignore comment and an explanation of the exception to the rule "any is bad"

Reply:
Changed in https://github.com/DataBiosphere/terra-ui/pull/3468/commits/24e1001ee4ba6b223b013965c647945e9b2c72b6.